### PR TITLE
Create narrow mode for Composer

### PR DESCRIPTION
**Pull request by gsouquet**
Fixes vector-im/element-web#18533
There will be a smaller diff once #6677 gets merged

Blocked by @janogarcia , currently awaiting confirmation of how the UI should be, this pull request creates an arbitrary breakpoints and a simple UI. The look is still being decided, and only minor changes should follow.

Screen Shot 2021-08-25 at 18 02 28
Screen Shot 2021-08-25 at 18 02 34

---
Here's what your changelog entry will look like:

## ✨ Features
 * Create narrow mode for Composer ([\#6682](https://github.com/matrix-org/matrix-react-sdk/pull/6682)). Fixes vector-im/element-web#18533.

Preview: https://6135c4b7988a0277d8747064--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by gsouquet** [comment]
@turt2live am I right to assume that your previous comment should be approve?

File: src/components/views/rooms/VoiceRecordComposerTile.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ export default class VoiceRecordComposerTile extends React.PureComponent<IProps,
     public render(): ReactNode {
         if (!this.state.recordingPhase) return null;
 
-        let stopOrRecordBtn;
+        let stopBtn;
         let deleteButton;
         if (!this.state.recordingPhase || this.state.recordingPhase === RecordingState.Started) {
```
Comment (turt2live) [inline src/components/views/rooms/VoiceRecordComposerTile.tsx:222]: this condition is a bit pointless, and adds classes which aren't needed

Comment (turt2live) [reply src/components/views/rooms/VoiceRecordComposerTile.tsx:222]: I can't get at the lines to offer a github suggestion, but the diff I'm proposing is:

```diff
diff --git a/src/components/views/rooms/VoiceRecordComposerTile.tsx b/src/components/views/rooms/VoiceRecordComposerTile.tsx
index e172fb7f0d..9ab089efe5 100644
--- a/src/components/views/rooms/VoiceRecordComposerTile.tsx
+++ b/src/components/views/rooms/VoiceRecordComposerTile.tsx
@@ -219,20 +219,14 @@ export default class VoiceRecordComposerTile extends React.PureComponent<IProps,

         let stopBtn;
         let deleteButton;
-        if (!this.state.recordingPhase || this.state.recordingPhase === RecordingState.Started) {
-            const classes = classNames({
-                'mx_MessageComposer_button': !this.state.recorder,
-                'mx_MessageComposer_voiceMessage': !this.state.recorder,
-                'mx_VoiceRecordComposerTile_stop': this.state.recorder?.isRecording,
-            });
-
+        if (this.state.recordingPhase === RecordingState.Started) {
             let tooltip = _t("Send voice message");
             if (!!this.state.recorder) {
                 tooltip = _t("Stop recording");
             }

             stopBtn = <AccessibleTooltipButton
-                className={classes}
+                className="mx_VoiceRecordComposerTile_stop"
                 onClick={this.onRecordStartEndClick}
                 title={tooltip}
             />;
```

**Review by turt2live — COMMENTED** [review]
otherwise looking good :D just a small amount of code cleanup

**Review by turt2live — APPROVED** [review]
Thank you! Apologies for the round trips :s

**Comment by turt2live** [comment]
this was still pending design review, so reverting as that probably wasn't the intention to land right away.

**Pull request merged**